docs/topics/Community_Resources.md

```diff
@@ -10,34 +10,34 @@ Many of these libraries are represented in the [unofficial, community-driven Dis
+| [discordcr](https://github.com/discordcr/discordcr)          | Crystal    |                                               |
+| [Discord.Net](https://github.com/RogueException/Discord.Net) | C#         |                                               |
+| [DSharpPlus](https://github.com/DSharpPlus/DSharpPlus)       | C#         | [Invite](https://discord.gg/KeAS3pU)          |
+| [dscord](https://github.com/b1naryth1ef/dscord)              | D          |                                               |
+| [DiscordGo](https://github.com/bwmarrin/discordgo)           | Go         | [Invite](https://discord.gg/0f1SbxBZjYq9jLBk) |
+| [DisGord](https://github.com/andersfylling/disgord)          | Go         | [Invite](https://discord.gg/0f1SbxBZjYq9jLBk) |
+| [catnip](https://github.com/mewna/catnip)                    | Java       | [Invite](https://discord.gg/yeF2HpP)          |
+| [Discord4J](https://discord4j.com/)                          | Java       | [Invite](https://discord.gg/NxGAeCY)          |
+| [Javacord](https://github.com/Javacord/Javacord)             | Java       | [Invite](https://discord.gg/0qJ2jjyneLEgG7y3) |
+| [JDA](https://github.com/DV8FromTheWorld/JDA)                | Java       | [Invite](https://discord.gg/0hMr4ce0tIk3pSjp) |
+| [discord.js](https://github.com/discordjs/discord.js)        | JavaScript | [Invite](https://discord.gg/bRCvFy9)          |
+| [Eris](https://github.com/abalabahaha/eris)                  | JavaScript |                                               |
+| [Discord.jl](https://github.com/Xh4H/Discord.jl)             | Julia      |                                               |
+| [Discordia](https://github.com/SinisterRectus/Discordia)     | Lua        | [Invite](https://discord.gg/EzRYYDW)          |
+| [discordnim](https://github.com/Krognol/discordnim)          | Nim        |                                               |
+| [RestCord](https://www.restcord.com/)                        | PHP        | [Invite](https://discord.gg/0duG4FF1ElFGUFVq) |
+| [discord.py](https://github.com/Rapptz/discord.py)           | Python     | [Invite](https://discord.gg/r3sSKJJ)          |
+| [disco](https://github.com/b1naryth1ef/disco)                | Python     |                                               |
+| [discordrb](https://github.com/discordrb/discordrb)          | Ruby       |                                               |
+| [discord-rs](https://github.com/SpaceManiac/discord-rs)      | Rust       |                                               |
+| [Serenity](https://github.com/serenity-rs/serenity)          | Rust       | [Invite](https://discord.gg/9X7vCus)          |
+| [AckCord](https://github.com/Katrix/AckCord)                 | Scala      | [Invite](https://discord.gg/5UH627u)          |
+| [Sword](https://github.com/Azoy/Sword)                       | Swift      |                                               |
```

Comment:
Added aegis.cpp's server invite, and invites to DAPI channels for libs without support servers.
Maybe we should put different text for dapi links? idk

```suggestion
| [aegis.cpp](https://github.com/zeroxs/aegis.cpp)             | C++        | [Invite](https://discord.gg/w7Y3Bb8)          |
| [discordcr](https://github.com/discordcr/discordcr)          | Crystal    | [Invite](https://discord.gg/puaxPkK)          |
| [Discord.Net](https://github.com/RogueException/Discord.Net) | C#         | [Invite](https://discord.gg/jkrBmQR)          |
| [DSharpPlus](https://github.com/DSharpPlus/DSharpPlus)       | C#         | [Invite](https://discord.gg/KeAS3pU)          |
| [dscord](https://github.com/b1naryth1ef/dscord)              | D          |                                               |
| [DiscordGo](https://github.com/bwmarrin/discordgo)           | Go         | [Invite](https://discord.gg/0f1SbxBZjYq9jLBk) |
| [DisGord](https://github.com/andersfylling/disgord)          | Go         | [Invite](https://discord.gg/0f1SbxBZjYq9jLBk) |
| [catnip](https://github.com/mewna/catnip)                    | Java       | [Invite](https://discord.gg/yeF2HpP)          |
| [Discord4J](https://discord4j.com/)                          | Java       | [Invite](https://discord.gg/NxGAeCY)          |
| [Javacord](https://github.com/Javacord/Javacord)             | Java       | [Invite](https://discord.gg/0qJ2jjyneLEgG7y3) |
| [JDA](https://github.com/DV8FromTheWorld/JDA)                | Java       | [Invite](https://discord.gg/0hMr4ce0tIk3pSjp) |
| [discord.js](https://github.com/discordjs/discord.js)        | JavaScript | [Invite](https://discord.gg/bRCvFy9)          |
| [Eris](https://github.com/abalabahaha/eris)                  | JavaScript | [Invite](https://discord.gg/D8FNF2P)          |
| [Discord.jl](https://github.com/Xh4H/Discord.jl)             | Julia      | [Invite](https://discord.gg/ng9TjYd)          |
| [Discordia](https://github.com/SinisterRectus/Discordia)     | Lua        | [Invite](https://discord.gg/EzRYYDW)          |
| [discordnim](https://github.com/Krognol/discordnim)          | Nim        |                                               |
| [RestCord](https://www.restcord.com/)                        | PHP        | [Invite](https://discord.gg/0duG4FF1ElFGUFVq) |
| [discord.py](https://github.com/Rapptz/discord.py)           | Python     | [Invite](https://discord.gg/r3sSKJJ)          |
| [disco](https://github.com/b1naryth1ef/disco)                | Python     |                                               |
| [discordrb](https://github.com/discordrb/discordrb)          | Ruby       | [Invite](https://discord.gg/cyK3Hjm)          |
| [discord-rs](https://github.com/SpaceManiac/discord-rs)      | Rust       |                                               |
| [Serenity](https://github.com/serenity-rs/serenity)          | Rust       | [Invite](https://discord.gg/9X7vCus)          |
| [AckCord](https://github.com/Katrix/AckCord)                 | Scala      | [Invite](https://discord.gg/5UH627u)          |
| [Sword](https://github.com/Azoy/Sword)                       | Swift      | [Invite](https://discord.gg/99a3xNk)          |
```

Reply:
@advaith1 Just double checking, these invites are perma invites right?

Reply:
I just copied them from their GitHub repos, so probably